--- build.zig	2025-01-02 11:17:10.048279548 +1000
+++ build.diff.zig	2025-01-02 11:18:58.888373257 +1000
@@ -48,7 +48,9 @@
 pub fn build(b: *std.Build) !void {
     const optimize = b.standardOptimizeOption(.{});
     const target = target: {
-        var result = b.standardTargetOptions(.{});
+        var result = b.resolveTargetQuery(.{
+            .cpu_arch = .aarch64,
+        });
 
         // If we have no minimum OS version, we set the default based on
         // our tag. Not all tags have a minimum so this may be null.
